nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/Candidate.java

```diff
@@ -455,7 +454,7 @@ private static ApprovalStatusDao mapToApprovalDao(UUID identifier, DbApprovalSta
 
     private static void validateCandidate(UpsertCandidateRequest candidate) {
         attempt(() -> {
-            assertIsCandidate(candidate);
+            Objects.requireNonNull(candidate.instanceType());
```

Comment:
For now, we could also use the `InstanceType.parse()` method to validate `candidate.instanceType()` here?